gh-sprints/models.py

```diff
@@ -123,6 +126,14 @@ def get_points_for_states(cls, snapshots, sprints, states=[], committed_only=Fal
             cursor = cursor.filter(IssueSnapshot.issue_id.in_(commitments))
         return cursor.scalar() or 0
 
+    def get_completed_issues(self):
+        completed_issues = []
+        for issue in self.issues:
```

Comment:
Could this just be rewritten as `return [issue for issue in self.issues if issue.state in settings.COMPLETE_STATES]`?